web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -34,7 +28,7 @@
       </div>
 
       <!-- Dropdown row for each entity property -->
-      <div *ngFor="let entityProp of chosenEntity?.definition.properties; let i = index;" class="src-prop-container">
```

Comment:
Let's leave these selectors the same.   Wiki and QA already using the old one.  Less churn.

Reply:
Can you tell me exactly what to change back? I'm not sure what I did. :-)

Reply:
@wooldridge , it's the class name, the old one is src-prop-container. @clockworked247 , I think it's okay to have the new name source-prop-container so it's consistent and more readable? Wiki and QA can change the naming

Reply:
@clockworked247 sounds like @ayuwono doesn't need this changed?

Reply:
Ok, just update then wiki for me then.